src/In.ProjectEKA.DefaultHip/DataFlow/Collect.cs

```diff
@@ -46,6 +46,8 @@ private List<string> FindPatientData(DataRequest request)
         {
             try
             {
+                var serializedDataRequest = JsonConvert.SerializeObject(request);
```

Comment:
You might have to remove the keymaterials from logging in or make it debug level.